### PR TITLE
ENH add a piggy back migration for jpeg turbo

### PR DESCRIPTION
**Pull request by beckermr**
Reverts regro/cf-scripts#1603

### CODE REVIEW

**Comment by beckermr** [comment]
@hmaarrfk we can merge this when you are ready! Sorry again for my confusion!

**Comment by hmaarrfk** [comment]
no issue. i'll add it to the merge lits.

File: conda_forge_tick/auto_tick.py [hunk 1]
```diff
@@ -662,6 +663,8 @@ def add_rebuild_migration_yaml(
     ]
     if migration_name == "qt515":
         piggy_back_migrations.append(QtQtMainMigrator())
+    if migration_name == "jpeg9e":
```
Comment (hmaarrfk) [inline conda_forge_tick/auto_tick.py:666]:
```suggestion
    if migration_name == "jpeg10":
```

https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/4091#discussion_r1106627033

Comment (hmaarrfk) [reply conda_forge_tick/auto_tick.py:666]: Kind bump to apply this suggestion. This will allow the migration to start.

**Pull request merged**